RecoJets/JetProducers/python/QGTagger_cfi.py

```diff
@@ -1,11 +1,12 @@
 import FWCore.ParameterSet.Config as cms
 
 QGTagger = cms.EDProducer('QGTagger',
-  srcJets		= cms.InputTag('ak4PFJetsCHS'),
```

Comment:
In principle with the `fillDescriptions` (and with default values), the _cfi could be removed, no?

Reply:
I assume so yes. This `_cfi` has always been there for a long time but maybe because someone used it to define the `QGTaggerTask`.

Reply:
The only use I find is this: https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py#L244

Perhaps we can take this opportunity to replace it with an import+clone and define the task there?

Reply:
Yeah. Okay I can do it.

Reply:
Thanks!